src/services/gateways/aws.ts

```diff
@@ -184,7 +186,7 @@ type AWSConfig = {
 export const IASQL_EC2_TAG_NAME = 'IaSQL_Name';
 
 export class AWS {
-  private ec2client: EC2Client
+  private ec2client: EC2
```

Comment:
Changing the client here would not affect the current ec2 methods that are expecting the old client? I mean shouldn't this be:
```suggestion
  private ec2client: EC2Client
  private newEc2Client: EC2
```